src/tree_data.c

```diff
@@ -1624,9 +1617,10 @@ lyd_new_path_check_find_lypath(struct ly_path *path, const char *str_path, const
     return ret;
 }
 
-API LY_ERR
-lyd_new_path2(struct lyd_node *parent, const struct ly_ctx *ctx, const char *path, const void *value,
-        LYD_ANYDATA_VALUETYPE value_type, uint32_t options, struct lyd_node **new_parent, struct lyd_node **new_node)
+static LY_ERR
```

Comment:
I suppose there should be some doxygen even though it will just be a copy of the API functions.

Reply:
agreed